app/views/form/page.html.erb

```diff
@@ -63,12 +63,15 @@
       <%= f.hidden_field :interruption_page_referrer_type, value: @interruption_page_referrer_type %>
 
       <div class="govuk-button-group">
-        <% if !@page.interruption_screen? && if request.query_parameters["referrer"] != "check_answers" %>
+        <% if !@page.interruption_screen? && if request.query_parameters["referrer"] != "check_answers" && request.query_parameters["referrer"] != "duplicate_logs" %>
```

Comment:
if this is merged after https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1983 then it should also exclude the `duplicate_logs_banner` referrer as it stands

Reply:
Yup, just saw that!

Reply:
I've just merged the above PR so this extra referrer check will need to be in this PR please :) happy to pair if you like/want any clarification